saleor/checkout/complete_checkout.py

```diff
@@ -573,8 +667,24 @@ def complete_checkout(
     :raises ValidationError
     """
     checkout = checkout_info.checkout
-    channel_slug = checkout_info.channel.slug
-    payment = checkout.get_last_active_payment()
+
+    # For individual payments, checkoutComplete will process the payment,
+    # otherwise a specific paymentID has to be provided
+    incomplete_payments = [
+        p for p in get_active_payments(checkout) if not p.is_authorized
```

Comment:
The condition shouldn't be if p.charge_status==NOT_CHARGED? This can return payments with statuses like capture etc. Right?